script/options.js

```diff
@@ -14,6 +15,7 @@ const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'entry', type: String, defaultValue: null },
   { name: 'analyzer', type: Boolean, defaultValue: false },
   { name: 'host', type: String, defaultValue: 'localhost' },
+  { name: 'hostname', type: String, defaultValue: 'localhost' },
   { name: 'public', type: String, defaultValue: null },
```

Comment:
Actually I think we can make the change to the `host` property above and work with that.

It seems like if a `host` is explicitly passed, it should use that as the `hostname`. But if not, then it should derive it from the build-type.  If neither is passed, maybe assume `localhost`?